imgcat/imgcat/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/campoy/tools/imgcat"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Fprintf(os.Stderr, "usage:\n\t%s [image_path]*\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	for _, path := range os.Args[1:] {
+		if err := cat(path); err != nil {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
+		}
+	}
+}
+
+func cat(path string) error {
+	w, err := imgcat.New(os.Stdout, imgcat.Inline(true))
```

Comment:
s/w/wc

Reply:
why? I think `w` is good

Reply:
w for Writer, wc for WriteCloser is an established convention. It makes it more obvious that user need to close the writer without checking the variable's type.